server/src/remote.rs

```diff
@@ -22,6 +22,8 @@ use crate::Context;
 
 use std::convert::TryInto;
 
+use tracing::{debug, error, info, warn, Level, event, instrument, span};
```

Comment:
Imports should be grouped as in `catalog.rs` and `main.rs`.

Reply:
Fixed

Reply:
I am not sure this is fixed.

Reply:
You are absolutely right. Now it should be fixed.